web/src/beta/pages/Editor/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+
+import LeftPanel from "@reearth/beta/features/LeftPanel";
+import Navbar, { Tab } from "@reearth/beta/features/Navbar";
+import RightPanel from "@reearth/beta/features/RightPanel";
+import Visualizer from "@reearth/beta/features/Visualizer";
+import { styled } from "@reearth/services/theme";
+
+type Props = {};
+
+const tabs = ["scene", "story", "widgets", "publish"];
+
+const Editor: React.FC<Props> = () => {
+  const [sceneId, setSceneId] = useState<string | undefined>(undefined); // This will come from project query once beta project creation works
+  const [currentTab, setTab] = useState<Tab | undefined>();
+  const location = useLocation();
+
+  useEffect(() => {
+    const splitPathname = location.pathname.split("/");
```

Comment:
Can we use `useParams`?

Reply:
useParams will only get me the sceneId. But I also need the final /widgets, /story to know the tab

Reply:
Hmm, as I thought, I think we should use params to switch tab. And redirect to 404 page if tab can't find.

Reply:
[simplify Editor logic](https://github.com/reearth/reearth/pull/444/commits/c8a28b13d1ad825d2a741737e7e93bff98234ff5)